clis/ph-cli/src/commands/inspect.ts

```diff
@@ -0,0 +1,74 @@
+import { Command } from "commander";
+import path from "node:path";
+import { CommandActionType } from "../types.js";
+import { getProjectInfo } from "../utils.js";
+export const inspect: CommandActionType<
+  [
+    string,
+    {
+      debug?: boolean;
+    },
+  ]
+> = async (packageName, options) => {
+  if (options.debug) {
+    console.log(">>> command arguments", { options });
+  }
+
+  const projectInfo = getProjectInfo(options.debug);
+
+  if (options.debug) {
+    console.log("\n>>> projectInfo", projectInfo);
+  }
+
+  try {
+    const manifest = (await import(path.join(packageName, "manifest"))) as {
```

Comment:
You can import the manifest type from `document-model/document`: https://github.com/powerhouse-inc/powerhouse/blob/8a5d3e33d554f6628235e0798d3dcebca324167a/packages/document-model/src/document/types.ts#L384